src/tests/unit/tests/common/components/__snapshots__/guidance-links.test.tsx.snap

```diff
@@ -1,51 +1,157 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`GuidanceLinksTest linkComponentType is defined as ExternalLink 1`] = `
-"<span className="guidance-links">
-  <ExternalLink href="https://url1" onClick={[Function: onClick]}>
-    TEXT1
-  </ExternalLink>
-</span>"
+<DocumentFragment>
+  <span
```

Comment:
We should mock the `ExternalLink` component so we have less irrelevant HTML in the snapshot.